### PR TITLE
Complete accountUpdate()

### PR DESCRIPTION
**Pull request by DandilionLau**
* accountUpdate() now completed and tested
* add mock variables `mockVerified` and `mockCookie` in accountUpdate()
* ~~to resolve domain conflict, change `accounts/@me` to  `accounts/@update` for update~~
* add travis test script to test updateAccount()
* change oraclejdk to openjdk in travis

### CODE REVIEW

File: src/main/resources/api.yaml [hunk 1]
```diff
@@ -93,7 +93,8 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /accounts/@me:
+
+  /accounts/@update:
```
Comment (sleepingpig) [inline src/main/resources/api.yaml]: That's not restful. "PUT" has expressed that the API is for update. Don't put operation on URL.
I think your problem can be fixed by simply changing `@me` to `{id}`

**Comment by sleepingpig** [comment]
Did you do rebase upon upstream? It looks like there're many commits of previous PRs.

File: src/main/resources/api.yaml (hunk 1 above)
Comment (DandilionLau) [reply src/main/resources/api.yaml]: Thx for pointing out. Indeed @update sounds wired and expose the operation. Now it has been fixed. I change it back to the original design accounts/@me (it was the `$curl` problem in the previous test leading to the `conflict in domain` problem).

**Comment by DandilionLau** [comment]
> Did you do rebase upon upstream? It looks like there're many commits of previous PRs.

My bad. We didn't rebase upon upstream. Do I need to rebase now or after this PR being approved?

**Pull request merged**